SQL_module/SubjectForm/index.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+    Include "./db_manipulation.php";
+
+    function validateField(String $text, int $size, &$valid, &$errors){
+        $tmp = $_POST["$text"];
+        if(!$tmp){
+            $errors["$text"] = "Полето е задължително."; 
+        } elseif(strlen($tmp) > $size ){
+            $errors["$text"] = "Полето има максимална дължина $size символа.";
+        } else {
+            $valid[$text] = $tmp;
+        }        
+    }
+
+    $valid = array();
+    $errors = array();
+
+    define("MAX_LENGTH_TITLE", 128);
+    define("MAX_LENGTH_LECTURER", 128);
+    define("MAX_LENGTH_DESCRIPTION", 1024);
+    
+    if( $_POST ){
+        
+        validateField('title', MAX_LENGTH_TITLE, $valid, $errors);
+        validateField('teacher',MAX_LENGTH_LECTURER, $valid, $errors);
+
+        validateField('description',MAX_LENGTH_DESCRIPTION, $valid, $errors);
+
+        if(count($valid) == 3 ){
```

Comment:
You may want to count the errors. What would be the behavior of the application if we add another field and forget to update this magic number "3"?